### DISCUSSION TITLE
[Bug] CHIA SCAM!!!!!

### DISCUSSION THREAD

**Discussion by vazo222** [Chia-Network/chia-blockchain#10823] [Support]
### What happened?

changed the passphrase, stole 3 chia changed again, and still stealing chia. the security system is terrible! China scam

### Version

1.2.11-1.3.0

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by vazo222** [comment]
xch1sq68hqkthcpjrz0eddeesum9pg2chkpxhgygh7g45mdplkz6c66s8sx4ma 
xch15znuzuz4wsge536wkuvdafsx69vxsvk27x3pnhx3j9rhzsztrhxsnwcuwq 
block this 2 wallets!!! They stolen 97 chia!!!!!!!!

**Comment by scrutinously** [comment]
Most likely this thief has your 24 words, either from using an OG pool like hpool, from third party software you used, or from a chia fork. You changing the passphrase on your personal computer doesn't protect you from someone inputting your compromised 24 words on another computer.

**Comment by MojoWhaleCom** [comment]
I don't understand. The first wallet was probably compromised, like scrutinously said. Did you move the chia to the new wallet, after creating it (getting new passphrases) and the Chia was stolen from the new wallet again? That would mean your computer is compromised and the person got the new passphrase too.

Anyway, it is completely impossible to access your wallet without having the passphrase, so think about where you put it in the past (hpool, chia fork, confirm wallet seed scam).

**Comment by scrutinously** [comment]
> I don't understand. The first wallet was probably compromised, like scrutinously said. Did you move the chia to the new wallet, after creating it (getting new passphrases) and the Chia was stolen from the new wallet again? That would mean your computer is compromised and the person got the new passphrase too.
> 
> Anyway, it is completely impossible to access your wallet without having the passphrase, so think about where you put it in the past (hpool, chia fork, confirm wallet seed scam).

I would assume he's talking about the client passphrase feature that encrypts the private keys locally so that you can't open the client without it, not calling the 24 word mnemonic a passphrase.

**Comment by shaneo257** [comment]
the security of the software is only as strong as the users security practices
